src/jvm/test/resources/expected_logs/validation_function_failure.txt

```diff
@@ -1,36 +1,56 @@
 = Validation function validateWithError has failed =
-| ----------- |
-|  Thread 1   |
-| ----------- |
-| operation() |
-| ----------- |
-| operation() |
-| ----------- |
-| operation() |
-| ----------- |
+| --------------------------------- |
+|      Thread 1       |  Thread 2   |
```

Comment:
It should require only one thread to reproduce the error

Reply:
I added the second thread to check that the table is well-formatted in the case when it has more than 2 columns. I added this explanation to the test.